homeassistant/components/open_meteo/weather.py

```diff
@@ -78,3 +78,41 @@ def wind_bearing(self) -> float | str | None:
         if not self.coordinator.data.current_weather:
             return None
         return self.coordinator.data.current_weather.wind_direction
+
+    @property
+    def forecast(self) -> list[Forecast] | None:
+        """Return the forecast in native units."""
+        if self.coordinator.data.daily is None:
+            return None
+
+        forecasts: list[Forecast] = []
+        daily = self.coordinator.data.daily
+        for index, time in enumerate(self.coordinator.data.daily.time):
+
+            forecast = Forecast(
+                datetime=time.isoformat(),
+            )
+
+            if daily.weathercode is not None:
+                forecast["condition"] = WMO_TO_HA_CONDITION_MAP.get(
```

Comment:
Shouldn't this be `forecast[ATTR_FORECAST_CONDITION]` and such?
See https://github.com/home-assistant/core/blob/dev/homeassistant/components/weather/__init__.py#L42-L50

Reply:
I don't think there is much benefit from that, as it already is a typed dictionary.
The reason we used constants in the past, is mostly to reduce typos, which are now already guarded.